cobrawap/__main__.py

```diff
@@ -65,7 +65,7 @@ def get_parser():
     action="store_true",
     help="print additional logging information",
 )
-CLI.add_argument("-V", "--version", action="version")
+CLI.add_argument("-V", "--version", action="version", version="cobrawap 0.2.1")
```

Comment:
In line with our spelling of cobrawap as a tool with a capital C, I would suggest:
```suggestion
CLI.add_argument("-V", "--version", action="version", version="Cobrawap 0.2.1")
```